packages/babel-types/src/builders/typescript/createTSUnionType.ts

```diff
@@ -1,14 +1,15 @@
 import { tsUnionType } from "../generated";
 import removeTypeDuplicates from "../../modifications/typescript/removeTypeDuplicates";
+import * as t from "../../types";
 
 /**
  * Takes an array of `types` and flattens them, removing duplicates and
  * returns a `UnionTypeAnnotation` node containing them.
  */
 export default function createTSUnionType(
-  typeAnnotations: Array<Object>,
-): Object {
-  const types = typeAnnotations.map(type => type.typeAnnotations);
+  typeAnnotations: Array<t.TSTypeAnnotation>,
+): t.TSType {
+  const types = typeAnnotations.map(type => type.typeAnnotation);
```

Comment:
looking at usage from babel-traverse apparently this was a typo and instead it should be `. typeAnnotation` instead